config.py

```diff
@@ -51,6 +55,10 @@ def __init__(
         hide_time_to_first_response: bool,
         ignore_user: List[str],
         labels_to_measure: List[str],
+        enable_mentor_count: str,
```

Comment:
```suggestion
        enable_mentor_count: bool,
```

gotta change this too. 😄